examples/chart/teleport/minikube.values.yaml

```diff
@@ -0,0 +1,19 @@
+proxy:
+  tls:
+    enabled: true
+
+extraVars:
+  # Provide the path to your own CA cert if you would like to use to
+  # validate the certificate chain presented by the proxy
+  SSL_CERT_FILE: "/var/lib/ca-certs/ca.pem"
```

Comment:
TLS_CERT_FILE -> SSL prefix is obsolete

Reply:
Really?!

I had been believing that you're relying on golang's feature to respect the SSL_CERT_FILE envvar to laod the alternative CA cert as seen in https://github.com/golang/go/search?q=SSL_CERT_FILE&type=Code

AFAIK, there is no reference to TLS_CCERT_FILE in golang's code-base. Am I missing something?

Reply:
I mean SSL prefix refers to an obsolete layer, but if you want to keep for compat reasons I'm ok with that

Reply:
I believe that I'm with you about the obsolete layer.

I'm just saying that rewriting `SSL_CERT_FILE` to `TLS_CERT_FILE` would make the example teleport installation unfunctional, because the support for the `TLS_CERT_FILE` is implemented nowhere?